### PR TITLE
UNITY-8128: adapt doc for new label

### PR DESCRIPTION
**Pull request by mjger**

### CODE REVIEW

File: _raml/apps/products/examples/get_product_productid_crossselling_response.json [hunk 1]
```diff
@@ -160,6 +160,7 @@
         "sfUrl": "http://pm.epages.com/epages/apidocu.sf/?ObjectPath=/Shops/apidocu/Products/Cust001",
         "productNumber": "Cust001",
         "isProductNumberVisible": true,
+        "isNew": false,
```
Comment (dkluever22) [inline _raml/apps/products/examples/get_product_productid_crossselling_response.json:163]: I'm not sure but I think this line needs to be added once more in this file after line 283.

File: apps/data-types.md [hunk 2]
```diff
@@ -577,6 +577,7 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | sfUrl | string | The link to the storefront URL of the product. |
 | productNumber | string | The product number. |
 | isProductNumberVisible | boolean | Indicates if the product number is displayed in the storefront. |
+| isNew | boolean | Indicates if the product should be displayed as in the storefront. |
```
Comment (dkluever22) [inline apps/data-types.md:580]: I would suggest to slightly change the wording. Furthermore I added the "word" as it was missing.
```suggestion
| isNew | boolean | Indicates if the product is displayed as new in the storefront. |
```

File: apps/data-types.md [hunk 3]
```diff
@@ -695,6 +696,7 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | sfUrl | string | The link to storefront URL of the product. |
 | productNumber | string | The product number. |
 | isProductNumberVisible | boolean | Indicates if the product number is displayed in the storefront. |
+| isNew | boolean | Indicates if the product should be displayed as new in the storefront. |
```
Comment (dkluever22) [inline apps/data-types.md:699]: I would suggest a slightly different wording:
```suggestion
| isNew | boolean | Indicates if the product is displayed as new in the storefront. |
```

**Review by dkluever22 — CHANGES_REQUESTED** [review]
@mjger I left you a few comments. Furthermore, I'm not entirely sure but I think that the example _raml/apps/search/post_search_products_response.json needs to be adjusted as well by adding **"isNew": false,** after line 108.
Please take a look and let me know if you have any questions :) Thanks!

File: apps/data-types.md (hunk 2 above)
Comment (mjger) [reply apps/data-types.md]: @dkluever22 you are totally right it must also be adapted in '_raml/apps/search/post_search_products_response.json'

**Review by dkluever22 — APPROVED** [review]

**Pull request merged**